src/panels/scene-viewer/SceneViewer.tsx

```diff
@@ -172,6 +175,10 @@ export const SceneViewer = (props: SceneViewerPropsFromParent) => {
       dataBindingTemplate[undecoratedKey] = selectedPropertyValue;
     }
 
+    if (!activeCamera || activeCamera.trim().length === 0) {
+      activeCamera = props.options.customInputActiveCamera;
```

Comment:
looks like you are using the same option to input either a template variable name, or an actual camera name which could be confusing for customers.. I'd suggest follow the same pattern as the selected E/C/P options, so this input is the variable name only. The default active camera value can be set to the template variable itself if needed

Reply:
Good call